src/collections/_documentation/performance-monitoring/getting-started.md

```diff
@@ -8,7 +8,7 @@ After this quick setup, you'll have access to [Performance](/performance-monitor
 {% capture __alert_content -%}
 Sentry's Performance features are currently in beta. For more details about access to these features, feel free to reach out at [performance-feedback@sentry.io](mailto:performance-feedback@sentry.io).
 
-Supported SDKs for Tracing
+Supported SDKs
 - JavaScript Browser SDK ≥ 5.16.0
```

Comment:
@HazAT It might be a good idea to bump up the minimum SDK versions that support tracing. I think the older versions have support for tracing features that have since been updated or changed.

Reply:
I will add our magic variables here to make sure ppl always use the latest.

Reply:
Then this section will be pointless, because it'll always point to the latest haha